cmd/crictl/stats.go

```diff
@@ -158,36 +156,40 @@ func ContainerStats(client pb.RuntimeServiceClient, opts statsOptions) error {
 	}
 	sort.Sort(containerStatsByID(r.Stats))
 
-	w := tabwriter.NewWriter(os.Stdout, 20, 1, 3, ' ', 0)
-	// Use `+` to work around go vet bug
-	fmt.Fprintln(w, "CONTAINER\tCPU %"+"\tMEM\tDISK\tINODES")
-	for _, s := range r.GetStats() {
-		id := getTruncatedID(s.Attributes.Id, "")
-		cpu := s.GetCpu().GetUsageCoreNanoSeconds().GetValue()
-		mem := s.GetMemory().GetWorkingSetBytes().GetValue()
-		disk := s.GetWritableLayer().GetUsedBytes().GetValue()
-		inodes := s.GetWritableLayer().GetInodesUsed().GetValue()
-		if !opts.all && cpu == 0 && mem == 0 {
-			// Skip non-running container
-			continue
-		}
-		old, ok := oldStats[s.Attributes.Id]
-		if !ok {
-			// Skip new container
-			continue
-		}
-		var cpuPerc float64
-		if cpu != 0 {
-			// Only generate cpuPerc for running container
-			duration := s.GetCpu().GetTimestamp() - old.GetCpu().GetTimestamp()
-			if duration == 0 {
-				return fmt.Errorf("cpu stat is not updated during sample")
+	display := newTableDisplay(20, 1, 3, ' ', 0)
+	for range time.Tick(500 * time.Millisecond) {
```

Comment:
Could we add a new option for this, e.g. `--watch` or `-w`?

Reply:
I think live monitoring is enabled by default, like `docker stats`. So do we still need this watch option?

Reply:
WDYT? @feiskyer Do we really need this option? What is the use case of non live monitoring?

Reply:
just like kubectl top, it only show the output for one time. We can enable it by default, but it could be disabled by an option

Reply:
Thanks for your reply, I have added this -w option to enable live monitoring, PTAL.